app/controllers/application_controller.rb

```diff
@@ -247,6 +247,19 @@ def show_offseason_banner?
     true
   end
 
+  def setup_sortable_client
+    @sort_column = clients_sort_column
+    @sort_order = clients_sort_order
+  end
+
+  def clients_sort_order
+    %w[asc desc].include?(params[:order]) ? params[:order] : "desc"
+  end
+
+  def clients_sort_column
+    %w[preferred_name id updated_at locale].include?(params[:column]) ? params[:column] : "id"
+  end
```

Comment:
If this is used by a few Controllers, but not most, would it feel cleaner to put this in a `controllers/concerns/client_sortable.rb` module or something similar that can be mixed in to different controllers?

Reply:
good idea!